src/Strategy/Node/Pnpm/PnpmLock.hs

```diff
@@ -156,7 +189,7 @@ instance FromJSON PnpmLockfile where
             then Map.insert "." virtualRootWs importers
             else importers
 
-    pure $ PnpmLockfile refinedImporters packages rawLockFileVersion
+    pure $ PnpmLockfile{importers = refinedImporters, packages = packages, lockFileVersion = rawLockFileVersion, lockFileSnapshots = snapshots}
     where
       getVersion (TextLike ver) = case (listToMaybe . toString $ ver) of
```

Comment:
I'm a bit confused about how `TextLike` deals with a version like '9.0', like we see [here](https://github.com/vuejs/core/blob/main/pnpm-lock.yaml#L1). Does that get parsed as an int and then turned into a string of `'9'`?

Reply:
Good call out, this was tricky and took me a bit of time to figure out. It turns out `listToMaybe` either returns `Nothing` for an empty list or the first element of the list in `Just` for a non-empty one.